native/src/components/Header.tsx

```diff
@@ -124,6 +127,17 @@ const Header = ({
     }
   }
 
+  const navigateToFeedback = () => {
+    if (cityCode) {
+      createNavigateToFeedbackModal(navigation)({
+        routeType: route.name as RouteType,
+        language: languageCode,
+        cityCode,
+        slug: (route.params as RoutesParamsType[PoisRouteType]).slug,
```

Comment:
Why as `PoisRouteType`? This might e.g. break feedback for the root category

Reply:
i added a function to define to correct slug for each route